doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
<p>"However, the main purpose of having a model deployed in the database is <strong>a</strong> getting quick response for a batch input. "</p><p> </p><p>missing commas: "For example, if we wanted to get the best answer for each row of the input table <code>MY_TEXT_TABLE</code><strong>, </strong>where text with a masked word is in the column <code>MY_TEXT_COLUMN</code><strong>, </strong>the SQL would look similar to this: "</p><p> </p><p>these also need to be changed in the other notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>